configuration/messageproperties/messages_zl_en.properties

```diff
@@ -118,62 +118,83 @@ ui.i18n.Location.name.58f3e355-ff28-4151-bbf6-5be3cb06fba8=CDI Klinik Ekstèn Je
 ui.i18n.Location.name.be50d584-26b2-4371-8768-2b9565742b3b=Achiv Santral
 ui.i18n.Location.name.4bd72351-80bb-4872-89d2-036ab77c2828=Achiv Santral
 ui.i18n.Location.name.1c470c79-91d1-4d7a-a1de-24b115d54384=Achiv Santral
+ui.i18n.Location.name.e0ff0e85-9c2f-4b14-88b1-17c26e84f2c1=Achiv Santral
```

Comment:
This is getting a little duplicative, but this is how we make sure that the display for each locatoin does not include the "| La Colline" or "| Thomonde" or whatever that is found in the full location name.